reframe/frontend/loader.py

```diff
@@ -96,10 +103,14 @@ def load_from_module(self, module):
             )
 
         if not hasattr(module, '_rfm_gettests'):
+            getlogger().debug('no tests registered')
```

Comment:
Usually in `getlogger().debug` you start with a capital letter the message. I don't have a preference, I am just commenting in case this was by accident
```suggestion
            getlogger().debug('No tests registered')
```
I am not sure at what point this message is printed but should we add the module in the message? Unless something else was printed above it and the message is not alone